jest.config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   },
   testURL: `http://localhost`,
   setupFiles: [`<rootDir>/loadershim.js`],
+  setupFilesAfterEnv: ['<rootDir>/setupTests.js'],
```

Comment:
@kevee I changed this [per the enzyme docs](https://enzymejs.github.io/enzyme/docs/installation/).

Reply:
👍